### PR TITLE
Added Descriptions For Commands

### PR DESCRIPTION
**Pull request by jonathanc-n**
I added descriptions with the help of LLM. I tried my best to fact check using Redis documentation and the existing PRs from  version notes.

References: https://github.com/apache/kvrocks/issues/2492

### CODE REVIEW

File: package.json
Comment (tisonkun) [inline package.json:1]: Please retain this meta to ensure the build is reproducible.

File: yarn.lock
Comment (tisonkun) [inline yarn.lock:1]: Touch content only should not modify the lockfile.

Comment (jonathanc-n) [reply yarn.lock:1]: Thanks for letting me know. Should be good now!

**Review by tisonkun — APPROVED** [review]
Thanks! Let's move forward.

File: docs/supported-commands.md [hunk 1]
```diff
@@ -55,126 +55,126 @@
...
+| SINTERSTORE | ✓                | v1.0.0        | Computes the intersection of multiple sets and stores the result in a destination set.                       |
+| SISMEMBER   | ✓                | v1.0.0        | Checks if a member is part of a set.                                                                         |
+| SMEMBERS    | ✓                | v1.0.0        | Returns all the members in a set.                                                                            |
+| SMOVE       | ✓                | v1.0.0        | Moves a member from one set to another.                                                                      |
+| SPOP        | ✓                | v1.0.0        | Pop members in the order of keys      |
+| SRANDMEMBER | ✓                | v1.0.0        | Returns one or more random members from a set without removing them. (always first N members if not changed) |
+| SREM        | ✓                | v1.0.0        | Removes one or more members from a set.                                                                      |
```
Comment (mapleFU) [inline docs/supported-commands.md:97]: I remember this is O(N) in kvrocks now

**Pull request merged**